requirements.txt

```diff
@@ -1,8 +1,8 @@
-allennlp>=2.3.0,<=2.9.0.dev20211215
+allennlp==2.10.0
```

Comment:
Can't we relax this to `>=` ?

Reply:
Just `allennlp>=2.10.0` feels too open. Maybe with `<2.11` as well?

Reply:
Good idea, also it is the common practice.

Reply:
Done.